src/cadence/contracts/MetadataViews.cdc

```diff
@@ -0,0 +1,131 @@
+/*
+This contract implements the metadata standard proposed
+in FLIP-0636.
+
+Ref: https://github.com/onflow/flow/blob/master/flips/20210916-nft-metadata.md
+
+Structs and resources can implement one or more
+metadata types, called views. Each view type represents
+a different kind of metadata, such as a creator biography
+or a JPEG image file.
+*/
+
+pub contract MetadataViews {
```

Comment:
Probably not super important, but this contract seems out of date(missing Royalty view for example)